person.rb

```diff
@@ -2,14 +2,21 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] For similar reasons mentioned previously Getters and Setters for `@rentals` should be present so kindly add it.